ios/Classes/SwiftOneginiPlugin.swift

```diff
@@ -80,6 +80,13 @@ func toOWCustomInfo(_ info: ONGCustomInfo?) -> OWCustomInfo? {
 }
 
 public class SwiftOneginiPlugin: NSObject, FlutterPlugin, UserClientApi, ResourceMethodApi {
+    func startApplication(securityControllerClassName: String?, configModelClassName: String?, customIdentityProviderConfigs: [OWCustomIdentityProvider]?, connectionTimeout: Int64?, readTimeout: Int64?, completion: @escaping (Result<Void, Error>) -> Void) {
+        OneginiModuleSwift.sharedInstance.startOneginiModule(httpConnectionTimeout: connectionTimeout) { result in
+            completion(result.mapError { $0 })
+        }
+    }
+
+

```

Comment:
Doesn't swift lint complain about two new lines? Please remove one.

Reply:
we don't have swiftlint set up yet for flutter, theres a story in current sprint to add it.